backend/src/test/java/de/neuefische/teddyscoronadiaries/controller/CovidControllerTest.java

```diff
@@ -29,6 +30,7 @@
 import static org.mockito.Mockito.when;
 
 @SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
+@TestPropertySource(properties = {"security.jwt.secret=supertestsecret"})
```

Comment:
you could also add a default value like 
``
 @Value("${security.jwt.secret:supertestsecret}")
    private String jwtSecret;
``
then you dont have to add it to a test where you dont need it